src/components/attributeTable/ToggleButton.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+
+  <v-btn icon :dark="dark" @click="toggleUi">
+    <v-icon medium>{{icon}}</v-icon>
+    {{text}}
+  </v-btn>
+
+</template>
+
+<script>
+import Vue from 'vue'
+import { WguEventBus } from '../../WguEventBus'
+import AttributeTableWin from './AttributeTableWin';
+
+export default {
+  name: 'wgu-attributetable-btn',
+  components: {
+    'wgu-attributetable': AttributeTableWin
+  },
+  props: {
+    // alternative icons: table_chart, view_comfy
+    icon: {type: String, required: false, default: 'table_chart'},
+    text: {type: String, required: false, default: ''},
+    dark: {type: Boolean, required: false, default: false}
+  },
+  data: function () {
+    return {
+      moduleName: 'wgu-attributetable'
+    }
+  },
+  created () {
+    var me = this;
```

Comment:
Not necessary since you are using the arrow functions below. Please use `this` instead.

Reply:
done